budgetportal/webflow/views.py

```diff
@@ -60,12 +67,50 @@ def provincial_infrastructure_project_detail(request, id, slug):
         % (snapshot.name, snapshot.province),
         "page_description": "Provincial infrastructure project by the %s %s department."
         % (snapshot.province, snapshot.department),
+        "csv_download_url": project.csv_download_url,
```

Comment:
this should be in the page_data variable so that it shows up in the embedded json for javascript to use
![Screenshot_2020-03-04_17-06-09](https://user-images.githubusercontent.com/235801/75893445-7c513680-5e3b-11ea-9f63-3d3ba6635bdf.png)

Reply:
Makes sense to me